WordPress/Classes/ViewRelated/Site Creation/Web Address/DesignSystem/Colors.xcassets/Foundation/Theme/brandJetpack.colorset/Contents.json

```diff
@@ -23,9 +23,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0x1F",
-          "green" : "0xB4",
-          "red" : "0x2F"
+          "blue" : "31",
+          "green" : "180",
+          "red" : "47"
```

Comment:
❓ Curious why the `brandJetpack.colorset` file has changed? I'm assuming Xcode automatically changed these values when you opened the file, is that correct?

Reply:
Not automatically. I changed the input method to 8-bit, so I can compare it to other colors easier. I'll revert the changes to reduce noise.

Reply:
Reverted in ee850c910c3fc15e4fc68689fe7af0d6c916395e